src/js/components/channels/twilio-channel-content.jsx

```diff
@@ -140,9 +146,17 @@ export class TwilioChannelContentComponent extends Component {
     }
 }
 
-export const TwilioChannelContent = connect((state) => {
+export const TwilioChannelContent = connect(({app, ui: {text}}) => {
     return {
-        ...state.integrations.twilio,
-        userId: state.user._id
```

Comment:
https://github.com/smooch/smooch-js/pull/417/files#r86180606
Oh, I guess we were always passing in `smoochId` to `ChannelContent` components?